rubailo_s/lesson9/homework.rb

```diff
@@ -0,0 +1,50 @@
+require_relative "./classes/person"
+require_relative "./classes/deck"
+
+players = [Person.new('John', 'john@test.rs', '777-7777-777'), Person.new('Mike', 'mike@test.rs', '444-4444-444')]
+
+players[0].greet(players[1])
+players[1].greet(players[0])
+
+first_player = players.sample
+second_player = players.reject {|p| p == first_player}.first
+
+puts "#{first_player.name} makes first move."
+
+deck = Deck.new
+deck.shuffle!
+
+while deck.cards_count > 0 do 
+	first_player_card = deck.draw
+	second_player_card = deck.draw
+
+	puts "#{first_player.name} gets #{first_player_card.point} #{first_player_card.suit}"
+	puts "#{second_player.name} gets #{second_player_card.point} #{second_player_card.suit}"
+
+	if first_player_card.point < second_player_card.point
```

Comment:
as an alternative
```rb
winner = if first_player_card.point > second_player_card.point
   first_player
elsif if first_player_card.point < second_player_card.point
  second_player
end
if winner
  winner.hand.add_cards # may be allow passing >1 cards?
else
   # returns cards back into deck
end 
```
You do the same actions, the only person changes. (what about using a method?)

The same approach can be used below

Reply:
I've changed code, it looks better.
Method add_card was changed to add_cards, we can send more than one card to player's hand.

> (what about using a method?)
Do you mean put winner selection logic into a method?
I've added such method in last commit